scripts/globals/spells/bluemagic/feather_storm.lua

```diff
@@ -24,22 +24,23 @@ end
 function onSpellCast(caster,target,spell)
     local params = {}
     -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
-        params.tpmod = TPMOD_CRITICAL
-        params.damageType = tpz.damageType.PIERCING
-        params.scattr = SC_LIGHT
-        params.numhits = 1
-        params.multiplier = 1.25
-        params.tp150 = 1.25
-        params.tp300 = 1.25
-        params.azuretp = 1.25
-        params.duppercap = 17
-        params.str_wsc = 0.0
-        params.dex_wsc = 0.0
-        params.vit_wsc = 0.0
-        params.agi_wsc = 0.30
-        params.int_wsc = 0.0
-        params.mnd_wsc = 0.0
-        params.chr_wsc = 0.0
+    params.tpmod = TPMOD_CRITICAL
+    params.attackType = tpz.attackType.PHYSICAL
```

Comment:
Do we want a different identifier for ranged physical spells?

Reply:
These aren't labeled as range in a very noticeable spot. But seems like the following 3 are classified as ranged:

Queasyshroom
Feather Storm
Pinecone Bomb

Reply:
should be changed to ranged yes

Reply:
Done.